tools/docker-network/run.sh

```diff
@@ -39,36 +40,34 @@ fi
 echo $DOCKER_BUILD_CONTEXT $DOCKERFILE_PATH
 docker compose -f $DOCKER_COMPOSE_FILE build --build-arg WITH_GO_WORK=${WITH_GO_WORK:-0} --build-arg DOCKER_BUILD_CONTEXT=${DOCKER_BUILD_CONTEXT} --build-arg DOCKERFILE_PATH=${DOCKERFILE_PATH}
 
-docker compose pull inx-indexer inx-blockissuer inx-faucet
+docker compose pull inx-indexer # inx-blockissuer inx-faucet
```

Comment:
why? :D

Reply:
I disabled it temporarily to have cleaner Docker logs and at the time there were some problems with those as they were not compatible with some other changes. It's still a draft so I'll revert it before marking it as ready.

Reply:
Sorry I saw that it's a draft after I finished the review 😅